FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -1329,6 +1329,12 @@ friend class FEXCore::IR::PassManager;
     }
   }
 
+#define DeriveOp(Dest, NewOp, Expr) \
+  if (ImplicitFlagClobber(NewOp)) \
+    SaveNZCV(); \
+  auto Dest = Expr; \
+  Dest.first->Header.Op = NewOp
```

Comment:
Macro hygiene to preempt accidental misuse:

```suggestion
#define DeriveOp(Dest, NewOp, Expr) \
  if (ImplicitFlagClobber((NewOp))) \
    SaveNZCV(); \
  auto Dest = (Expr); \
  Dest.first->Header.Op = (NewOp)
```

Reply:
fixed, thx